app/api/src/docker/serverless.yml

```diff
@@ -24,60 +24,30 @@ provider:
         file: ./cadquery/Dockerfile
   apiGateway:
     metrics: true
-    binaryMediaTypes:
-      # we need to allow binary types to be able to send back images and stls, but it would be better to be more specific
-      # ie image/png etc. as */* treats everything as binary including the json body as the input the lambdas
-      # which mean we need to decode the input bode from base64, but the images break with anything other than */* :(
-      - '*/*'
```

Comment:
Removing this magically fixed CORS issues I was having as I mentioned in a previous comment.